src/components/calcite-shell/usage/advanced.md

```diff
@@ -0,0 +1,115 @@
+#### Advanced
+
+Renders a shell with leading and trailing floating panels, action bar/pad, block, flow, tip manager, footer
+
+```html
+<calcite-shell>
+  <calcite-shell-panel slot="primary-panel" layout="leading" detached>
+    <calcite-action-pad id="action-pad" placement="side" slot="action-pad" hidden>
```

Comment:
actionpad doesn't have `placement`. If its hidden, can we remove it from the code?

Reply:
that one is hidden bc it pops up on click of the upper right icon in the shell panel. I can take it out all together?

Reply:
i think taking it out is easier. i just removed it. better not to clutter it up any more since it's already a lot of code